frontends/verilog/verilog_parser.y

```diff
@@ -112,6 +112,10 @@ struct specify_rise_fall {
 
 %define api.prefix {frontend_verilog_yy}
 
+%glr-parser
+%expect 21
+%expect-rr 2
```

Comment:
Are we sure that those 21 shift-reduce conflicts are all justified? Is there anything to gain from using `%expect` if the number is that large? (I'm assuming that it's okay to have conflicts without `%expect` statement in GLR mode.)

Reply:
How are we sure that these 21 shift-reduce conflicts are harmless? Did you actually check each of those?
Is it possible to silence individual shift-reduce warnings just to be sure we won't be covering new conflicts that might appear in the future?
I think I've tried GLR in the past when I worked on my private branch trying to add typedefs, but gave up on all those shift-reduce. I was able to restructure the parser a bit and reduce the number of conflicts, but not eliminate them entirely. I came to the conclusion that the {...} code sections within the rules is one of the reasons for these conflicts because you cannot look ahead past them, or something like that.
I haven't put time into fully understanding why the earlier approach (doing symbol lookup at the lexer) was ruled out by packages.

Reply:
I haven't tried to work out the conflicts, I'm not sure if there is a good way to investigate them? AIUI, in the worst case a conflict will produce a run time error rather than a mis-parse so they should easily be caught (as was the case with SV interfaces before). 

The `%expect` was mostly added to suppress a warning that may confuse people building Yosys and make sure we know of any new conflicts added from now on.

imo, doing resolution in the lexer is going to make it very difficult to handle scoping correctly